samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -9,12 +9,37 @@
 
 from samcli.cli.cli_config_file import configuration_option, TomlProvider
 from samcli.cli.main import pass_context, common_options, aws_creds_options, print_cmdline_args
+from samcli.commands.pipeline.bootstrap.oidc_config import (
+    BitbucketOidcConfig,
+    GitHubOidcConfig,
+    OidcConfig,
+    GitLabOidcConfig,
+)
+from samcli.commands.pipeline.bootstrap.pipeline_oidc_provider import (
+    BitbucketOidcProvider,
+    GitHubOidcProvider,
+    GitLabOidcProvider,
+    PipelineOidcProvider,
+)
 from samcli.lib.config.samconfig import SamConfig
-from samcli.lib.pipeline.bootstrap.stage import Stage
+from samcli.lib.pipeline.bootstrap.stage import (
+    BITBUCKET_REPO_UUID,
+    DEPLOYMENT_BRANCH,
+    GITHUB_ORG,
+    GITHUB_REPO,
+    GITLAB_GROUP,
+    GITLAB_PROJECT,
+    OIDC_CLIENT_ID,
+    OIDC_PROVIDER,
+    OIDC_PROVIDER_URL,
+    PERMISSIONS_PROVIDER,
+    OIDC_SUPPORTED_PROVIDER,
```

Comment:
This variable `OIDC_SUPPORTED_PROVIDER` is not been used in command parameters. Just to clarify my earlier comment, please move any imports that is not used until line 269 into that line.

Reply:
Got it. Thanks for catching it. Updated the imports